### ISSUE TITLE
Export Cables Must Be Whitelisted

### ISSUE THREAD

**Issue by Lightning11wins** [PrinceOfAmber/Storage-Network#550]
Export cables are _always_ filtered by whitelist. Sometimes, though, the player may want to not filter at all and export all items. A toggle to filter by blacklist, similar to the one in the filtered import cable, would be greatly appreciated.

**Comment by Dima-Lisovvenko** [comment]
yes i am the player. Please, i mean filtered import already has it, shouldn't be hard, right?

**Comment by Dima-Lisovvenko** [comment]
it wasnt, awaiting pull, but i am not the only one, so in the meantime try this -- https://github.com/Dima-Lisovvenko/Storage-Network/tree/build-v1.20.1. build/libs should have .jar, just import into mods. if it doesnt hit me up pls
